cooltools/dotfinder.py

```diff
@@ -1741,21 +1739,25 @@ def clustering_step(
     (to be tested).
 
     """
+    # Annotate regions, if needed:
+
+    scores_df = scores_df.copy()
+    if (
+        not "region" in scores_df.columns
+    ):  # If input scores are not annotated by regions:
+        scores_df["region"] = np.where(
+            scores_df["chr1"] == scores_df["chrom2"], scores_df["chrom1"], np.nan
```

Comment:
@agalitsyna don't `"chr1"` - looks suspicious here ... should it be `chrom1` - for cis ?

Reply:
also `np.where` signature looks different `np.where(condition, [x,y])` ...

Reply:
Need to check that !

Reply:
np.where can work similarly to ternary operator on vectors, pretty cool: https://numpy.org/doc/stable/reference/generated/numpy.where.html